wmonk/create-react-app-typescript
# Update Jest config so `.js` and `.jsx` are transformed correctly

* `.js` and `.jsx` should be transformed by `ts-jest` to make sure any ES6+ features are transpiled
* ~~The `fileTransform.js` should not be run on any JavaScript or TypeScript files~~ [resolved by 31e3ffea78fe7993a86281d0fae260917b27c942]

Fixes #136, and changes order and regex style to more closely match `create-react-app`.

How to verify that these changes fix the issue:

* Create a `.tgz` containing the modified `react-scripts` package
* Serve the `.tgz` over HTTP(S)
* Create a new project, but set `--scripts-version` to the path to the `.tgz`
* Follow the "steps to reproduce" from #136. The test should now pass.
